.github/workflows/sanity.yml

```diff
@@ -0,0 +1,30 @@
+name: "nightly sanity test"
+on:
+  schedule:
+    - cron:  '30 20 * * 1-5'
+
+jobs:
+  sanity:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-2016]
+        version: [8, 11, 14]
+        impl: [hotspot, openj9]
+        buildlist: [openjdk, system, functional]
+    steps:
+    - uses: actions/checkout@v1
+    - uses: AdoptOpenJDK/install-jdk@v1
+      with:
+        version: ${{ matrix.version }}
+        targets: JDK_${{ matrix.version }}
+        impl: ${{ matrix.impl }}
+        source: 'nighlty'
```

Comment:
spelling -> nightly instead of nighlty

Reply:
Thanks for pointing it out! My testing is using default releases so didn't catch it up. Updated.